### DISCUSSION TITLE
5주차 회고

### DISCUSSION THREAD

**Discussion by Seoin02** [Together-3team/petFrontend#148] [회고]
### TRY(PROBLEM의 해결책으로 다음 회고까지 실행할 점)
아래 양식을 복사해 댓글로 작성해주시면 됩니다.

```
### KEEP

### PROBLEM

### TRY
```

**Comment by devwqc** [comment]
### KEEP
- 카톡, 디스코드 등을 통해서 빠르게 소통하고 있다.
- 프로젝트 막바지인데 다들 열심히 하고 있다.

### PROBLEM
- 이번 주부터 API 연동 작업이 들어가야 하는데 홈, 상품 목록 페이지가 아직 시작되지 않았다.

### TRY
- 남은 기간 집중해서 얼른 끝내고 다른 팀원들 도와주자!

**Comment by Seoin02** [comment]
### KEEP
- 본인이 맡은 부분을 열심히 한다.
- BE, FE, 디자이너가 소통이 원활하고, 피드백 반영이 빠르다.
- 서로 잘한 부분을 칭찬하는 문화가 작업할 때 힘이 된다.
- 멘토님이 작업에 도움되는 피드백을 많이 해주신다.

### PROBLEM
- 

### TRY
- 차주에 API 작업 마무리 해봅시다!

**Comment by minyoung0503** [comment]
### KEEP
격려와 칭찬을 아낌없이 해준다.
소통이 빠르다.
회의 내용을 깔끔하게 정리해주신다.

### PROBLEM
API 연결이 미뤄지고 있다.

### TRY
